sql-schema/999.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE `customoids` ( `customoid_id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY, `device_id` int(11) NOT NULL DEFAULT '0', `customoid_descr` varchar(255) NULL DEFAULT '', `customoid_deleted` tinyint(1) NOT NULL DEFAULT '0', `customoid_current` double NULL, `customoid_prev` double NULL, `customoid_oid` varchar(255) NULL, `customoid_unit` varchar(20) NULL, `customoid_divisor` int(11) NOT NULL DEFAULT '1', `customoid_multiplier` int(11) NOT NULL DEFAULT '1', `customoid_limit` double NULL, `customoid_limit_warn` double NULL, `customoid_limit_low` double NULL, `customoid_limit_low_warn` double NULL, `customoid_alert` tinyint(1) NOT NULL DEFAULT '0', `customoid_passed` tinyint(1) NOT NULL DEFAULT '0', `lastupdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, `user_func` varchar(100) NULL );
```

Comment:
Setting this to 999 will cause many issues.  For one, users who have tested this without reverting their database by hand will not get schema updates until we reach schema 1000...